misc/language-tour-guide/src/content/6-processes/processes.mdx

```diff
@@ -0,0 +1,9 @@
+export const frontmatter = { order: 1 };
+
+# Processes
+
+In Elixir, all code runs inside processes. Processes are isolated from each other, run concurrent to one another and communicate via message passing. Processes are not only the basis for concurrency in Elixir, but they also provide the means for building distributed and fault-tolerant programs.
+
+Elixir's processes should not be confused with operating system processes. Processes in Elixir are extremely lightweight in terms of memory and CPU (even compared to threads as used in many other programming languages). Because of this, it is not uncommon to have tens or even hundreds of thousands of processes running simultaneously.
+
+In this chapter, we will learn about the basic constructs for spawning new processes, as well as sending and receiving messages between processes.
```

Comment:
Without any code, this looks quite empty. Maybe you could put here something like

```elixir
IO.puts "Continue to the next page"
```